code/mob/living/critter/flock/flockdrone.dm

```diff
@@ -210,32 +197,20 @@
 	if (!src.flock)
 		..()
 		return
-
 	src.flock.hideAnnotations(src)
-
+	//hold a ref to the controller so we can put them back with the flock
+	var/mob/living/intangible/flock/old_controller = src.controller
 	if (src.controller)
+		src.release_control_abrupt(FALSE)
```

Comment:
```suggestion
		src.release_control(FALSE)
```

Reply:
you said you wanted dormantize to not beep, `release_control` beeps